lib/vanagon/component.rb

```diff
@@ -290,6 +290,14 @@ def get_build_dir
       end
     end
 
+    def get_dependency_hash
+      obj = @options
+      %i[version name].each do |var|
+        obj[var.to_s] = self.instance_variable_get("@#{var}")
```

Comment:
I don't think we should metaprogram this kind of stuff.

Metaprogramming is really only useful when you don't know what "the string" is, like something that's input from the user/outside.

Just access the 2 variables directly.